crates/red_knot_python_semantic/resources/mdtest/type_of/dynamic.md

```diff
@@ -1,13 +1,20 @@
 # `type[Any]`
 
+This file contains tests for non-fully-static `type[]` types, such as `type[Any]` and
+`type[Unknown]`.
+
 ## Simple
 
 ```py
-def f(x: type[Any]):
+def f(x: type[Any], y: type[str]):
     reveal_type(x)  # revealed: type[Any]
     # TODO: could be `<object.__repr__ type> & Any`
     reveal_type(x.__repr__)  # revealed: Any
 
+    z = x
+    x = y
+    y = z
```

Comment:
This is OK, but I feel like this would be easier to read and understand (for me, anyway):
```suggestion
def f(x: type[Any], y: type[str]):
    reveal_type(x)  # revealed: type[Any]
    # TODO: could be `<object.__repr__ type> & Any`
    reveal_type(x.__repr__)  # revealed: Any

    # type[str] and type[Any] are assignable to each other
    a: type[str] = x
    b: type[Any] = y
```

Reply:
ah yeah, that's much better. Thanks!